packages/entities/entities-shared/src/components/entity-empty-state/EntityEmptyState.vue

```diff
@@ -163,6 +174,16 @@ const { i18n: { t } } = composables.useI18n()
 const useCanCreate = ref(false)
 const showCreateButton = computed((): boolean => useCanCreate.value && !!props.actionButtonText)
 
+const EmptyStateAppearance = computed((): AppearanceTypes | [AppearanceTypes, string] => {
```

Comment:
```suggestion
const emptyStateAppearance = computed((): AppearanceTypes | [AppearanceTypes, string] => {
```

Don't capitalize variables. Only classes, types, interfaces, etc should be capitalized.